spec/classes/backend_resources_spec.rb

```diff
@@ -4,181 +4,275 @@
   on_supported_os({facterversion: '3.8.0'}).each do |os, facts|
```

Comment:
There's no logic wrapped around this class. Having this loop over every supported platform seems like it is just adding a lot of testing time without real benefit. Suggest setting this to EL7.

Reply:
Limited those tests and ones for `sensu::backend::default_resources` to just EL7 since both are really just defining types with no OS specific logic.